fix broken links in website docs

lovettchris opened the pull request:
<!-- Thank you for submitting a pull request! -->
<!-- ⚠️⚠️ Do Not Delete This! pull_request_template ⚠️⚠️ -->
<!-- Please read our contribution guidelines: https://microsoft.github.io/AirSim/CONTRIBUTING/ -->

Fixes: #    <!-- add this line for each issue your PR solves. -->
lots of broken links found with a webcrawler tool.

## About
Fixing broken links.
This is extracted from the much bigger PR# 3546 so I can break that PR into easier to review parts.

## How Has This Been Tested?
mkdocs build and the webcrawler tool.

## Screenshots (if appropriate):

Commit:
fix broken links.

Commit:
add missing info

zimmy87 commented on docs/px4_sitl.md:
```diff
@@ -155,25 +178,28 @@ NOTE: Do `NOT` do this on a real drone as it is too dangerous to fly without the
 
-There are several options for flying the simulated drone using a remote control or joystick like xbox gamepad. See [remote controllers](remote_control.md#RC_Setup_for_PX4)
+There are several options for flying the simulated drone using a remote control or joystick like
+xbox gamepad. See [remote controllers](remote_control.md#RC_Setup_for_PX4)
```
I think the right link for this page is: remote_control.md#rc-setup-for-px4

lovettchris replied:
yes indeed, thanks for catching this!

zimmy87 commented on docs/unreal_upgrade.md:
```diff
@@ -16,7 +16,7 @@ These instructions apply if you are already using AirSim on Unreal Engine 4.16.
 1. From your AirSim repo folder, run 'clean_rebuild.sh`.
 2. Rename or delete your existing folder for Unreal Engine.
-3. Follow step 1 and 2 to [install Unreal Engine 4.24](https://github.com/Microsoft/AirSim/blob/master/docs/build_linux.md#install-and-build).
+3. Follow step 1 and 2 to [install Unreal Engine 4.24](build_linux.md#install-and-build).
```
I don't see any #install-and-build section in build_linux.md, does that part of the link still need to be there?

lovettchris replied:
Yep, good catch, I'll remove the fragment.

zimmy87 reviewed (changes requested):
Looks good for the most part, but I think a few links are wrong and some confusing text elsewhere.

Commit:
Merge branch 'master' into clovett/brokenlinks

lovettchris commented on docs/px4_setup.md:
````diff
@@ -43,15 +41,18 @@ See also [initial firmware setup video](https://dev.px4.io/starting-initial-conf
     }
 ```
 
+The "Barometer" setting keeps PX4 happy because the default AirSim barometer has a bit too much
````
Ah, yes, this part is waiting on my other PR "mavlinkfixes"... I'll remove this sentence until that PR is in.

lovettchris commented on docs/px4_sitl.md:
````diff
@@ -53,19 +63,21 @@ The default ports have changed recently, so check them closely to make sure AirS
     ```
-    Notice the PX4 `[simulator]` is using TCP, which is why we need to add: `"UseTcp": true,`.
+    Notice the PX4 `[simulator]` is using TCP, which is why we need to add: `"UseTcp": true,`. The
+    "Barometer" setting keeps PX4 happy because the default AirSim barometer has a bit too much
````
Fixed

Commit:
code review feedback.

lovettchris commented on docs/log_viewer.md:
```diff
@@ -2,26 +2,66 @@
 
-If you can't build the LogViewer.sln, there is also a [click once installer](http://www.lovettsoftware.com/LovettSoftware/Downloads/Px4LogViewer/Px4LogViewer.application).
+If you can't build the LogViewer.sln, there is also a [click once
+installer](http://www.lovettsoftware.com/LovettSoftware/Downloads/Px4LogViewer/Px4LogViewer.application).
```
Thanks for the excellent review, I've fixed all the issues you found.

rajat2004 reviewed (commented):
`distance_sensor.md` should be added to the navigation in `mkdocs.yml` so that it appears in the sidebar. `mkdocs build` does give an info message about the missing page, but currently doesn't treat it as a warning

Commit:
add to toc

lovettchris commented:
@rajat2004, fixed and I added a toc level for "Sensors" to group this content as I expect it will grow over time...
![image](https://user-images.githubusercontent.com/18707114/115499540-74fa1180-a224-11eb-8873-d2631936b887.png)
